rosidl_generator_dds_idl/cmake/rosidl_generate_dds_interfaces.cmake

```diff
@@ -54,6 +55,8 @@ macro(rosidl_generate_dds_interfaces target)
         list(APPEND _generated_msg_files "${_output_path}/${_name}_.idl")
       elseif(_parent_folder STREQUAL "srv")
         list(APPEND _generated_srv_files "${_output_path}/${_name}_.idl")
+      elseif(_parent_folder STREQUAL "action")
```

Comment:
Do we also want another `elseif` block in the parent scope for the file extension? And maybe more importantly if a .srv file appears inside the an action folder? E.g.

```cmake
elseif(_extension STREQUAL ".srv")
  if(_parent_folder STREQUAL "srv")
    list(APPEND _generated_srv_files "${_output_path}/Sample_${_name}_Request_.idl")
    list(APPEND _generated_srv_files "${_output_path}/Sample_${_name}_Response_.idl")
  elseif(_parent_folder STREQUAL "action")
    list(APPEND _generated_action_files "${_output_path}/Sample_${_name}_Request_.idl")
    list(APPEND _generated_action_files "${_output_path}/Sample_${_name}_Response_.idl")
  else()
    ...
  endif()
elseif(_extension STREQUAL ".action")
  # Do something here?
else()
  ...
endif()
```